### PR TITLE
Repaths evidence bags to storage

### LINKED ISSUE #9544: If you put a primed flashbang (any other grenade?) inside an evidence bag, it "stays" there after detonation.
Basically, prime a flashbang, put it inside the evidence bag. After it detonates, grenade is still in the bag, in a primed state (blinking lights). Trying to empty the bag doesn't get rid of it, gotta put something inside and then empty it to get rid of flashbang icon.

Minor thing, yet still a bug, so reporting it here.

### PR DESCRIPTION
**Pull request by Cyberboss**
Fixes #9544

Needs balancing

### CODE REVIEW

**Comment by jughu** [comment]
Friend

File: code/modules/detectivework/evidence.dm [hunk 1]
```diff
@@ -1,94 +1,72 @@
...
+
+/obj/item/weapon/storage/evidencebag/proc/RecalculateWeightClass(mob/user)
+	var/old_wc = w_class
+	w_class = WEIGHT_CLASS_TINY
+	for(var/A in contents)
+		var/obj/item/I = A
+		if(istype(I))
+			w_class = max(w_class, I.w_class + 1)
+	if(user)
+		var/diff = abs(old_wc - w_class)
```
Comment (coiax) [inline code/modules/detectivework/evidence.dm]: Instead of all this abs() stuff, why not a switch() statement with -3,-2,-1,1,2,3

**Comment by Cyberboss** [comment]
@optimumtact so quick with the WIP tag of death as always

**Comment by optimumtact** [comment]
you said it needed testing not me

**Comment by optimumtact** [comment]
or balancing rather not testing

**Comment by optimumtact** [comment]
tick tock

**Comment by Cyberboss** [comment]
F U C K I T

**Pull request closed without merging**